### PR TITLE
Fmi news 2024 03

### PR DESCRIPTION
**Pull request by chrbertsch**
@ all FMI-Designers: Please add your topics and add content

### CODE REVIEW

File: _2024-03/fmi-news.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+{::options parse_block_html="true" /}
+
+![FMI](FMI.png)
+
+#### First FMI Advisory Committee meetting
+
+After the clarification of FMI Project membership roles, the and Project hosted the first FMI Advisory Committee meeting on Nov 5th 2025.
+ We received very valuable feedback, ideas and feature requests!
+Thanks to the 70 participants for their contributions and participation!
+(Participating Companies: Akkodis, Ansys, AVL, Beckhoff, Boeing, Bosch, Claytex, COMSOL, DassaultSystems, dSpace, EKSINTEC, GMMotorsports, KEB, KnorrBremse, LBL, LTX, MachineWare, MapleSoft, Mathworks, ModelbasedCloud, Modelon, NVIDIA, PMSF, PrincipiaMBS, Renault, SamaresEngineering, Siemens, Synopsys, Volkswagen, VolvoAutonomousSolutions)
```
Comment (bmenne-dspace) [inline _2024-03/fmi-news.md:16]: dSpace = dSPACE

File: _2024-03/fmi-news.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+As more and more modelling and simulation tools extend their support for FMI 3.0, now advanced features such as synchronous clocks get more into the focus. 
+Thanks to Cláudio Gomes for providing a very instructive example with several FMUs on Github (https://github.com/clagms/synchronous-clock-fmus), that can be used to understand the concept and validate the synchronous clock support if importing tools. 
+Already successfully tested in Altair Activate by Masoud Najafi (Thanks!).
+
+![Synchronous-clocls](Synchronous-clocks.png)
+
+#### Current development of layered standards
+
+FMI Layered Standard for for Network Communication (FMI-LS-BUS) making progress!
+The version v1.0.0 for CAN, CAN FD, CAN XL is in beta testing currently, and will be demonstrated at the InternationASAMConference in Munich in December.
```
Comment (bmenne-dspace) [inline _2024-03/fmi-news.md:49]: InternationASAMConference = Internation ASAM Conference

File: _2024-03/fmi-news.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+And in parallel we made progress on the Flexray support and are happy that tody we could tag v1.1.0alpha
```
Comment (bmenne-dspace) [inline _2024-03/fmi-news.md:50]: Shall "tody" be "today" (if it is the case I would just leave the word out)

File: _2024-03/fmi-news.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+Currently an inspection period is running for the release candidates, which you can find here together with a summary of changes: https://github.com/modelica/fmi-standard/releases
+In case of any findings, please open an issue https://github.com/modelica/fmi-standard/issues a.s.a.p.!
+
+#### Examples of SynchronousClocked FMUs available
+
+As more and more modelling and simulation tools extend their support for FMI 3.0, now advanced features such as synchronous clocks get more into the focus. 
+Thanks to Cláudio Gomes for providing a very instructive example with several FMUs on Github (https://github.com/clagms/synchronous-clock-fmus), that can be used to understand the concept and validate the synchronous clock support if importing tools. 
+Already successfully tested in Altair Activate by Masoud Najafi (Thanks!).
+
+![Synchronous-clocls](Synchronous-clocks.png)
```
Comment (bmenne-dspace) [inline _2024-03/fmi-news.md:44]: clocls = clocks?

Comment (clagms) [reply _2024-03/fmi-news.md:44]: support if importing tools = support in importing tools

File: _2024-03/fmi-news.md (hunk 2 above)
Comment (clagms) [reply _2024-03/fmi-news.md:49]: International

File: _2024-03/fmi-news.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+
+#### FMI Design Face2Face Meeting in Munich Dec 2-3 2024
+
+AVL will host a face-2-face in Munich at the beginning of December.
+We will work on layered standards to the FMI standard and future improvements.
+
+#### ASAM Conference in Munich Dec 4-5 2024
+
+The FMI Projekt will present "FMI, Layered Standards and ASAM Standards - Enabling Seamless SiL Simulation of Virtual ECUs" at the coming International ASAM Conference in Munich.
```
Comment (clagms) [inline _2024-03/fmi-news.md:63]: FMI Project

File: _2024-03/fmi-news.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+![ASAM-Conference](asam-conference.jpg)
+
+#### 16th Internation Modelica and _FMI_ Conference!
+
+We are are happy to invite you to the coming https://modelica.org/events/modelica2025/, where the importance of FMI is now also reflected in the conference title!
+
+
+#### Other Resources and Discussion Forums for FMI related Questions
+
+* Join the [LinkedIn FMI community](https://www.linkedin.com/groups/7477473/) to get the latsted news on FMI, FMI supporting tools and discussions within the user community.
```
Comment (clagms) [inline _2024-03/fmi-news.md:85]: latest

**Review by clagms — CHANGES_REQUESTED** [review]

**Pull request merged**